lib/vsc/administration/vo.py

```diff
@@ -355,12 +386,29 @@ def _set_member_quota(self, storage_name, path, member, quota):
         @type quota: integer (hard value)
         """
         try:
-            hard = quota * 1024 * self.storage[self.host_institute][storage_name].data_replication_factor
-            soft = int(hard * self.vsc.quota_soft_fraction)
+            storage = self.storage[self.host_institute][storage_name]
+        except KeyError:
+            errmsg = "Trying to access non-existent field %s in the storage dictionary"
+            logging.exception(errmsg, storage_name)
+        else:
+            fs_backend, fs_backend_err = storage.load_operator()
```

Comment:
looks like this methdo will have to return the fs_backend_err too

Reply:
indeed, totally changed to use the new `StorageOperator` class in vsc-filesystems. See https://github.com/hpcugent/vsc-administration/pull/134/commits/557c3681738c7694cc80cfbef58535b6d0325e83